python/scripts/test_leak.py

```diff
@@ -57,4 +57,5 @@ def leak2():
 
         gc.collect()
 
+
```

Comment:
As a sidenote, an idea for a sprint task: investigate what this script does and whether it still exercises a real leak.